AirCasting/SessionViews/ChartViewModel.swift

```diff
@@ -8,10 +8,78 @@ import Combine
 import SwiftUI
 
 final class ChartViewModel: ObservableObject {
+    enum Session {
```

Comment:
we need to add a task to unify (meaning creating a protocol and conforming both to it) sessions before this gets out of hand